web-test-runner.config.mjs

```diff
@@ -0,0 +1,19 @@
+// import { rollupBundlePlugin } from '@web/dev-server-rollup';
+import { esbuildPlugin } from '@web/dev-server-esbuild';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+// import resolve from "@rollup/plugin-node-resolve"
+// import typescript from "@rollup/plugin-typescript"
+
+export default {
+  browsers: [
+    playwrightLauncher({ product: 'chromium' }),
+    playwrightLauncher({ product: 'firefox' }),
+    playwrightLauncher({ product: 'webkit' }),
+  ],
+  nodeResolve: true,
+  files: "./src/tests/unit/**/*_tests.ts",
+
```

Comment:
Here we can add:

```js
  testFramework: {
    config: {
      ui: "tdd"
    }
  },
```

That configures mocha to use [the TDD interface](https://devhints.io/mocha-tdd) - `test(...)` instead of `it(..)` - whic is more consistent with the functional tests.

Reply:
Perfect! Thanks!